web/src/user_topics.ts

```diff
@@ -18,6 +18,13 @@ import {get_time_from_date_muted} from "./util";
 
 export type ServerUserTopic = z.infer<typeof user_topic_schema>;
 
+export type AllVisibilityPolicies = {
+    INHERIT: 0;
+    MUTED: 1;
+    UNMUTED: 2;
+    FOLLOWED: 3;
+};
```

Comment:
This seems like a somewhat strange type, duplicating the actual value of this? 

It's not a lot to duplicate, and seems like a strange pattern, but maybe it's good for this use case anyway.

Reply:
The numeral-literals used in the type (0, 1, ...)  are intented to make the type more strict ie. so that we can not use numbers other than `{0, 1, 2, 3}`. Though the commit was not needed as we already have a `AllVisibilityPolicies`.  Also, we can do something like

```typescript
type VisibilityPolicy = "INHERIT" | "MUTED" | "UNMUTED" | "FOLLOWED" 
type AllVisibilityPolicies = {
  [key in VisibilityPolicy]: number
}
```
if the latter look enough strange ??